subprojects/android-test/ui-testing-core-app/src/androidTest/kotlin/com/avito/android/ui/test/BitmapMatcherTest.kt

```diff
@@ -53,5 +54,6 @@ class BitmapMatcherTest {
         assertThat(error).hasMessageThat().contains("expected=#FF0000FF")
     }
 
-    private fun getDrawable(@DrawableRes id: Int): Drawable = rule.activity.getDrawable(id)!!
+    private fun getDrawable(@DrawableRes id: Int): Drawable =
+        InstrumentationRegistry.getInstrumentation().targetContext.getDrawable(id)!!
```

Comment:
force unwrap to `requireNotNull() { "Can't find drawable by ${id to res name}" }

Reply:
`requireNotNull` throws `IllegalArgumentException`. I think `Resources.NotFoundException` is more convenient in this case.